src/layers/microscopy-viewer-layer/microscopy-viewer-layer-base.js

```diff
@@ -84,6 +85,7 @@ export class MicroscopyViewerLayerBase extends BaseTileLayer {
       colorValues: orderedColorValues,
       minZoom,
       getTileData,
+      // eslint-disable-next-line no-shadow
```

Comment:
Would it make the code less clear to avoid shadowing in this particular case?

Reply:
We'd have to change the name of the viewport util `getTileIndices` to something, and I don't mind the shadowing especially since it conveys what is actually happening.